solar_installation/demo/helpdesk_ticket.xml

```diff
@@ -0,0 +1,138 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+    <record id="helpdesk_ticket_7" model="helpdesk.ticket">
+        <field name="name">Raj Sharma's Ticket</field>
+        <field name="stage_id" ref="helpdesk.stage_solved"/>
+        <field name="answered_customer_message_count">1</field>
+        <field name="product_id" ref="product_product_10"/>
+        <field name="avg_response_hours">5.074</field>
```

Comment:
avg_response_hours, total_response_hours, first_response_hours, ... 
I don't think we need to define all those fields. They will be computed as soon as a mail.message is created, if I understand correctly